src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPlugin.groovy

```diff
@@ -104,7 +105,7 @@ class DockerJavaApplicationPlugin implements Plugin<Project> {
             description = 'Builds the Docker image for the Java application.'
             dependsOn createDockerfileTask
             conventionMapping.inputDir = { createDockerfileTask.destFile.parentFile }
-            conventionMapping.tag = { determineImageTag(project, dockerJavaApplication) }
+            conventionMapping.tags = { [determineImageTag(project, dockerJavaApplication)] as Set }
```

Comment:
Using the `tags` property of DockerBuildImage instead of (deprecated) property `tag` allows overriding the default config of `javaApplication`. In DockerBuildImage, `tag` has preference over `tags`, which made it effectless to set `tags` before.

I don't consider this change to be breaking, because the behavior is only changed if someone had the aforementioned effectless property `tags` set.